xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-plugins/src/main/webjar/xwiki-macro/macroWizard.css

```diff
@@ -45,8 +48,13 @@ ul.macro-parameter-group-members > li.more {
   font-size: smaller;
 }
 
+li.auto-macro-list .macro-description {
+  width: 0;
+  min-width: 100%;
```

Comment:
Would be good to add a comment explaining what you want to do this this.

Reply:
This might not be needed with https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-ui/src/main/resources/CKEditor/EditSheet.xml#L822 .

Reply:
Indeed, I didn't notice you had added this rule with Quick Actions.

Some of the badges wrap due to the addition of that rule. Do you think it's okay?
![image](https://github.com/xwiki/xwiki-platform/assets/132468278/195e9934-d1ff-4c02-9a30-98a8dc2bd951)

Maybe the max width rule should be specific to Quick Actions?